input/params/defaults.par

```diff
@@ -243,7 +243,7 @@ ReionSMParam_c         : 2.0
 ReionSMParam_d         : 2.5
 
 ReionTcool             : 1.0e4  # Kelvin
-ReionNionPhotPerBary   : 6000 # Since the IMF has been updated for the UV luminosity.
+ReionNionPhotPerBary   : 4000 
```

Comment:
see more in the conversation

Reply:
The value of 6000 is the one that Balu used in his fiducial simulation (I checked that in the log files of Balu's fiducial simulation). @s-balu can you confirm that?

Reply:
ah yes. you are right. i forgot to check values in my input.par... 

I will change it back. But I also noticed that this defaults.par does not have the correct, e.g. 

ReincorporationModel
ReincorporationEff  
ReionGridDim
SfEfficiency    
SfCriticalSDNorm
SnModel                   : 2
SnEjectionEff             : 1.53782     # SN feedback gas reheating efficiency
SnEjectionScaling         : 1.0
SnEjectionScaling2        : 3.2

SnReheatRedshiftDep       : 2.0
SnReheatEff               : 6.97996    # mass loading factor for SN feedback
SnReheatScaling           : 1.0
SnReheatScaling2          : 3.2
SnReheatNorm              : 60.0
SnReheatLimit             : 10000.0    # maximum allowed mass loading factor for SN feedback

@s-balu  can you plz confirm?

Reply:
Hi @qyx268 yes I just checked my input params and I had used 6000. That comment about the UV luminosity was added by me, so I remember

Reply:
Yes the values you have highlighted are indeed different. This is a difference between the L10 box and the L210 box (When I am running my simulations in the L210 box I change the parameters above)